tests/clickhouse-test

```diff
@@ -668,6 +690,7 @@ class TestCase:
 
             self.runs_count += 1
             self.testcase_args = self.configure_testcase_args(args, self.case_file, suite.suite_tmp_path)
+            client_options = SettingsRandomizer.add_random_settings(client_options)
```

Comment:
Maybe also worth applying them to `client_options_query_str`/`env['CLICKHOUSE_URL_PARAMS']` ?